src/api/index.tsx

```diff
@@ -80,7 +80,7 @@ export const makeApiURL = ({
 };
 
 export const makeQueryString = (query: {
-  [key: string]: string | number | null | void | boolean;
+  [key: string]: string | number | null | undefined | boolean;
```

Comment:
Really? Weird

Reply:
Yes. I think `undefined !== void` but more `undefined` is included in `void`. The failure comes from the `url` lib, which explicitly requires `undefined`, not `void`. I will file an issue to investigate this `void` vs `undefined` problem.